src/pokedex.c

```diff
@@ -4435,7 +4436,7 @@ bool16 HasAllMons(void)
     return TRUE;
 }
 
-void sub_80C09B0(u16 a)
+static void ResetOtherRegisters(u16 a)
```

Comment:
This function needs a more descriptive name.  Something like `ResetVideoRegisters()`.